lib/fittings/swagger_render.js

```diff
@@ -6,6 +6,7 @@ var injector = require('../../index.js').injector;
 var _ = injector.get('_');
 
 module.exports = function create(fittingDef) {
+    injector = require('../../index.js').injector;
```

Comment:
@yyscamper I'm not entirely sure why this works.  For some reason there is a difference in the injector object between initially requiring the fitting and loading the fitting.  Given that this is a temporary work-around for a test-only problem, I think it's acceptable for now.